src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -19,28 +11,7 @@
     {
         <script asp-src="~/OrchardCore.Lists/Scripts/ci-list-ordering.min.js" debug-src="~/OrchardCore.Lists/Scripts/ci-list-ordering.js" asp-name="ci-list-ordering" at="Foot" depends-on="Sortable, jQuery"></script>
     }
-
-    var authorizedContentTypeDefinitions = new List<ContentTypeDefinition>();
-
-    foreach (var contentTypeDefinition in Model.ContainedContentTypeDefinitions)
-    {
-        if (await AuthorizationService.AuthorizeAsync(
-            User,
-            CommonPermissions.EditContent,
-            await ContentManager.NewAsync(contentTypeDefinition.Name)))
-        {
-            authorizedContentTypeDefinitions.Add(contentTypeDefinition);
-        }
-    }
-    var dropdownClassList = new List<string>()
-    {
-        "dropdown-menu"
-    };
-
-    if (CultureInfo.CurrentUICulture.IsRightToLeft())
-    {
-        dropdownClassList.Add("dropdown-menu-start");
-    }
+  
```

Comment:
I am not sure we should remove this logic.

I think the `@if (!Model.ContainedContentTypeDefinitions.Any())`  should be `@if(authorizedContentTypeDefinitions.Count == 0)`

But I am not sure. One will have to look and see if this is indeed needed or can be removed.

Reply:
original pr #11150  . added authorization in correct place